eclipse/sw360
# feat(SplitComponentUI): Split Component data like releases and attachments from Source Component to Target Component

> Split Component data like releases and attachments from Source Component to Target Component
> * Which issue is this pull request belonging to and how is it solving it? (*#881*)
> * Did you add or update any new dependencies that are required for your change? - No

### How To Test?
> - Open a component which has releases or attachment. Click on split button.
> - Choose a target component
> - Move some releases or attachments from source component to target component
> - `Note: ` - User should have write access to both component in order to perform split , else split should be prevented, proper message should be displayed
> - Both source and target Component should not have active moderaion request -  , else split should be prevented, proper message should be displayed

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
